.pre-commit-config.yaml

```diff
@@ -36,25 +36,25 @@ repos:
             args: [--branch, master]
 
     - repo: https://github.com/jumanjihouse/pre-commit-hook-yamlfmt.git
-      rev: 0.0.10
+      rev: 0.2.2
       hooks:
           - id: yamlfmt
 
     - repo: https://github.com/jumanjihouse/pre-commit-hooks.git
-      rev: 2.1.6 # Do not update to 3.0.0 unless macOS comes with Ruby 2.7
+      rev: 3.0.0 # Do not update to 3.0.0 unless macOS comes with Ruby 2.7
```

Comment:
This shell be downgraded.

Reply:
Yep, automated upgrade. Will revert it back.

Reply:
Reverted back